.github/workflows/stash/action.yml

```
@@ -13,8 +13,11 @@ runs:
   using: "composite"
   steps:
     - name: Create stash
-      run: tar -acf ${{ runner.temp }}/${{ inputs.name }}.tar.zst -C ${{ inputs.path }} ./
+      run: tar -acf ${{ runner.temp }}/"{$ARTIFACT_NAME}".tar.zst -C "${ARTIFACT_PATH}" ./
```

```suggestion
      run: tar -acf "${{ runner.temp }}/${ARTIFACT_NAME}".tar.zst -C "${ARTIFACT_PATH}" ./
```

move quote to whole expression and fix typo on { position.